static/js/millennium_diff_table.js

```diff
@@ -0,0 +1,602 @@
+const conformanceURL = 'https://fhir-ehr.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/metadata';
+const conformanceHeaders = { Accept: 'application/json+fhir' };
+
+const capabilityStatementURL = 'https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/metadata';
+const capabilityStatementHeaders = { Accept: 'application/fhir+json' };
+
+// Configuration for resources that have been altered from DSTU2 to R4, or extended from Basic
+const resourceConfig = ({
+  combinedResources: [
+    {
+      resourcesCombined: ["Conformance"],
```

Comment:
Probably better, and to make it clearer, to change `resourcesCombined` to `dstu2Resources` and `combinedTo` to `r4Resources`. This is because technically a resource could be "split" from one into multiple from DSTU2 to R4.

Reply:
Updated here: b28a505288416745aca925fe2095e8e1d7e96628